lib/aur-query

```diff
@@ -19,7 +19,8 @@ uri_info() {
         else if (NR > 1)
             printf "&arg[]=%s", $0
     } END {
-        printf "\n"
+        if (NR != 0)
```

Comment:
`mapfile -t` strips the newline at the end so I guess this isn't needed. (Same if you use while/read and `[[ $foo ]] && bar+=("$foo")`

Reply:
This was to address the fact that `aur depends` sometimes calls aur-query without input, (because the entries were already 'seen').

The `uri_info` takes this empty input and unconditionally adds a trailing `\n`. And although mapfile strips the `\n` the 'empty string' remains in the array. Which is then converted into `url<space><nothing><nl>` making curl complain about the lack of a url.

`aur depends carla-bridges-win32-git` (at preset date) and `aur query -tinfo </dev/null ` will trigger this corner case.